http-client-spi/src/main/java/software/amazon/awssdk/http/HeaderValue.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.http;
+
+import software.amazon.awssdk.annotations.SdkProtectedApi;
+
+/**
+ * Constants for HTTP header values
+ */
+@SdkProtectedApi
+public class HeaderValue {
+
+    public static final String CHUNKED = "chunked";
```

Comment:
Can we put the header value in `Header` class above? It seems unnecessary to create another class for it to me.